indico/modules/events/surveys/fields/simple.py

```diff
@@ -55,19 +59,84 @@ class NumberField(SurveyField):
+    display_type = IndicoRadioField(_('Display type'), [DataRequired()],
+                                    description=_('Show the question either as radio button or select'),
+                                    choices=[('radio', _('Show as radio button')),
+                                             ('select', _('Show as select field'))],
+                                    default='radio')
+    radio_display_type = IndicoRadioField(_('Arrangment of available options'),
+                                          [HiddenUnless('display_type', 'radio')],
+                                          choices=[('vertical', _('Vertical alignment')),
+                                                   ('horizontal', _('Horizontal alignment'))])
+    options = MultipleItemsField(_('Options'), [DataRequired()],
+                                 fields=[('option', _('Option'))], unique_field='option',
+                                 description=_('Specify options available for selection by user'))
+
+
+class SingleChoiceField(SurveyField):
+    name = 'single_choice'
+    friendly_name = _('Single Choice')
+    config_form = SingleChoiceConfigForm
+    wtf_field_class = None
+
+    def create_wtf_field(self):
+        field_options = {}
+        if self.question.field_data['display_type'] == 'select':
+            self.wtf_field_class = SelectField
+        else:
+            self.wtf_field_class = IndicoRadioField
+            field_options['orientation'] = self.question.field_data['radio_display_type']
+        choices = [(iter['option'], iter['option']) for iter in self.question.field_data['options']]
+        return self._make_wtforms_field(self.wtf_field_class, choices=choices, **field_options)
+
+
+class TextParagraphConfigForm(IndicoForm):
+    _common_fields = {'title'}
+
+    title = StringField(_('Title'), [DataRequired()], description=_("The title of the field"))
+    content = TextAreaField(_('Content'), description=_('Text content displayed in the field'))
+
+
+class TextParagraphField(SurveyField):
+    name = 'text_paragraph'
+    friendly_name = _('Text Paragraph')
+    config_form = TextParagraphConfigForm
+    wtf_field_class = TextAreaField
+
+    def create_wtf_field(self):
+        return self._make_wtforms_field(self.wtf_field_class, default=self.question.field_data['content'])
+
+
+class MultiSelectConfigForm(FieldConfigForm):
+    choices = MultipleItemsField(_('choices'), [DataRequired()], fields=[('choice', _('Choice'))], unique_field='choice',
+                                 description=_('Specify choices available for selection by user'))
+
+
+class MultiSelectField(SurveyField):
+    name = 'multiselect'
+    friendly_name = _('Select multiple')
+    config_form = MultiSelectConfigForm
+    wtf_field_class = IndicoSelectMultipleCheckboxField
+
+    def create_wtf_field(self):
+        choices = [(iter['choice'], iter['choice']) for iter in self.question.field_data['choices']]
```

Comment:
`iter` is a builtin; don't use it as a loop var. why not `choice` or if you want something shorter simply `x` (but i'd go for `choice` here)